openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -142,7 +142,7 @@ def _generate_locked_out_error_message():
                                      'locked. Try again in {locked_out_period} minutes.\n'
                                      'To be on the safe side, you can reset your '
                                      'password {link_start}here{link_end} before you try again.\n')).format(
-            link_start=HTML('<a href="/reset" >'),
+            link_start=HTML('<a http="/login" class="form-toggle" data-type="password-reset">'),
```

Comment:
Please run the core unit tests for this app, I think tests will fail. Use `is_testing_env` flag for this change if tests are failing.

Reply:
No tests are failing because of the changes